outputs.tf

```diff
@@ -0,0 +1,23 @@
+output "ngw_id" {
+  value       = "${join("", aws_nat_gateway.default.*.id)}"
```

Comment:
that hack doesn't need here, you can use ` value       = ["${aws_nat_gateway.default.*.private_ip}"]`

Reply:
same for all outputs with `concat`

Reply:
`ngw_id` should be a string, not list.

Reply:
my mistake, I thought about NGW per subnet